h5c/vic/src/vic-webapp/e2e/vch-create-wizard/1-basic.e2e-spec.ts

```diff
@@ -194,12 +194,26 @@ describe('VCH Create Wizard - Basic', () => {
     });
   });
 
-  it('should delete created vch', () => {
+  it('should redirect to VCH VM and display Create Wizard menu item', () => {
+    page.navigateToVchVm(namePrefix + specRunId);
+    // wait for VM summary page to be ready
+    browser.wait(function () {
+      return browser.isElementPresent(by.cssContainingText('.summary-name-label', namePrefix + specRunId));
+    }, defaultTimeout * 6);
+    page.clickByCSS('.summary-action-link');
+    // wait for menu items to be calculated
+    browser.sleep(defaultTimeout);
+    page.clickByText('#applicationMenuContainer .k-item .k-link', 'All VIC Actions');
+    browser.sleep(defaultTimeout);
+    // TODO: go back to vch list
+  });
+
+  xit('should delete created vch', () => {
```

Comment:
so it looks like the above one, create, is working..but you have an xit on this one? Is this one still not working right?

Reply:
Just excluded those so you can test to there. The problem was that I can't click the sub-items of the VIC menu item, because when I move the mouse (through Protractor) to that menu item, the actions menu disappears.